### DISCUSSION TITLE
Deployment guide?

### DISCUSSION THREAD

**Discussion by tractatus** [GMOD/Apollo3#169] [Q&A] [answered]
Hi,

This looks very exciting.

Is there a deployment guide similar to:
https://github.com/GMOD/Apollo/blob/develop/docs/Setup.md

To help out getting Apollo3 setup and started? I was trying some basic npm and yarn commands to get it up and running but got stuck.

**Comment by garrettjstevens** [comment, marked as answer]
Hi, thanks for your interest in Apollo 3! It is still under development, but the current goal is to have a public beta release available in May of 2023 and we'll have a setup guide available at that time.

If you're interested, you can look at our [milestones](https://github.com/GMOD/Apollo3/milestones) for updates on our timeline.

**Reply by tractatus** [reply]
@garrettjstevens Great to hear! Having Apollo taking full advantage of JBrowse2 will be splendid and also creates lots of ways to add and extend.

Looking forward!
